source/site/scss/main.scss

```diff
@@ -1,37 +1,41 @@
 body {
-  background-image: none;
-  background-repeat: no-repeat;
-  background-size: cover;
-  background-position-y: center;
+  background-color: white;
   margin: 0;
 }
 
-*::selection {
-  background-color: mediumturquoise;
-}
-
 .navbar {
-  overflow: auto;
   background-color: white;
+  overflow: auto;
   position: fixed;
   width: 100%;
   top: 0;
   left: 0;
   font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  border-bottom: 1px solid grey;
 }
 
 .navbar a {
   float: right;
   display: block;
-  color: black;
-  text-align: center;
   padding: 18px 16px;
+  text-align: center;
   text-decoration: none;
+  color: 0F0F0F;
 }
 
 .navbar a:hover {
```

Comment:
Don't know if you've used it before, but you're writing CSS in a SCSS file here. That's fine, since SCSS is a superset of CSS (i.e. all CSS is valid SCSS). But to take advantage of it, you can nest styles in SCSS! So these navbar classes can be written as: 

```scss
.navbar {
  // `.navbar` styles
  ...
  a {
    // `.navbar a` styles
    ...
    &:hover {
      // `.navbar a:hover` styles
      ...
    }
  }
}
```
There's a bunch of other nice features that come with SCSS *hint hint*. But feel free to use whichever style you like